src/components/DateSelection.vue

```diff
@@ -28,36 +30,23 @@ export default {
 </script>
 
 <style>
-
+    div[role="button"] {
+        height: 8em;
+    }
+    div[role="button"]:hover, div[role="button"]:focus {
```

Comment:
Il y a un tout petit bug dans la gestion de la navigation au clavier.

Si je clique sur un bouton de sélection de date (qui passe au vert), et que j'appuie sur `tab`, le focus passe au bouton suivant (qui devient vert), donnant l'impression que le critère de recherche correspondant est actif, ce qui n'est pas le cas.

Je ne sais pas dans quelle mesure il s'agit d'un problème de style ou de comportement, mais j'ai l'impression qu'il faudrait distinguer les états `focus` et `actif`.

Reply:
j'ai bien noté que la nav clavier sur cette partie était pas optimale ;)
j'en garde pour demain !
(mais c'est bien le comportement normal de passer d'élément en élément avec la touche tab)